test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -85,17 +79,18 @@ int fillRandomTest(dev::test::TestSuite const& _testSuite, std::string const& _t
 	if (wasError)
 	{
 		std::cerr << json_spirit::write_string(v, true);
-		return 1;
+		return std::string();
 	}
+	else
+		std::cout << json_spirit::write_string(v, true);
 
-	std::cout << json_spirit::write_string(v, true);
-	return 0;
+	return json_spirit::write_string(v, true);
 }
 
 /// Parse Test string replacing keywords to fuzzed values
-void dev::test::RandomCode::parseTestWithTypes(std::string& _test, std::map<std::string, std::string> const& _varMap)
+void dev::test::RandomCode::parseTestWithTypes(std::string& _test, std::map<std::string, std::string> const& _varMap, RandomCodeOptions& _options)
```

Comment:
Can this be `RandomCodeOptions const&`?

Reply:
I guess no. if it use random generators it can't be const. and we want it to use both static (from input) and random code generation.

Reply:
Which field has to be modified?

Reply:
Why do you think it can't be const when it uses random generators?